### DISCUSSION TITLE
Batch Delete in All Pages View

### DISCUSSION THREAD

**Discussion by jsmorabito** [athensresearch/athens#1249] [Feature Requests]
Would like the ability to delete multiple pages at a time. Here is what it looks like in Roam:

![image](https://user-images.githubusercontent.com/46250921/120083537-58f55580-c097-11eb-97d4-04c0dd0a1f35.png)

![image](https://user-images.githubusercontent.com/46250921/120083555-6b6f8f00-c097-11eb-95e7-9cb1d17f08bd.png)

**Comment by sid597** [comment]
This is open in #554
